TeamMandate/Mandate.md

```diff
@@ -12,17 +12,13 @@ This will be done through an open collaborative approach that will further ESDC'
 
 ## Mandate (team's authority within IITB)
 
-- Manage IT Strategic Evolution Fund 
-- Full ownership and governance of projects operating against the IT Strategic Evolution fund
-- Provide recommendations to senior management on how IT can be leveraged to improve ESDC's mandate delivery 
-- Conduct both inward facing and external research in order to provide evidence based aforementioned recommendations 
-- Permeate and enforce modern IT practices and processes 
-- Provide strategic direction to IITB's initiatives in order to maximize their impacts and transition IITB into a holistic service-oriented organization
-- Produce IT strategies that includes diagnosis, guiding policies, and sets of coherent actions, for IITB to continually improve its responsiveness to business change
-- Channel senior level support to reprioritize tasks to align ESDC to the approved vision
-- Continuously monitor technology and policy trends to keep pace with technological development (such as OECD AI Policy Observatory)
-- Forge partnerships with key IM/IT partners and suppliers, including across jurisdictions and private and public institutions, for utilizing their expertise and knowledge to meet service delivery goals
-- Orchestrate the IT ecosystem to bring strategies to completion
+- Authority on Open Source documentation 
+- Report to senior management on digital strategic objectives of ESDC
+- Report to senior management on compliance with TBS directives 
+- Authority to merge working groups, or committees, within IITB
```

Comment:
Not really an action, might need to be something like: Identify and merge dubplicate committees, working groups, etc.

But then, I'm not interested in reviewing ToRs to see if there's some little difference that legitimitally supports having those two different but similar working groups?

Reply:
After discussion, we agreed that this doesn't needs to happen: the best groups will move out of the pack.

However, there needs to be a dimension of sharing the work. (work in the open.)